Makefile

```diff
@@ -1,16 +1,31 @@
 .DEFAULT_GOAL := help
 
-release: ## release a new version
-	@echo "Releasing a new version."
-	@echo "This is a remote release, it will push to the remote repository."
-	semantic-release --strict version --changelog --push --tag --commit
+install: ## install the dependencies
+	@echo "Installing dependencies."
+	@echo "Installing semantic-release."
+	pip install python-semantic-release
```

Comment:
python-semantic-release is part of the dev requirements so you simply install the package the same way the action does `pip install -e ".[dev]"`